presto-noop/src/main/java/io/prestosql/plugin/noop/NoOpPlugin.java

```diff
@@ -94,5 +103,23 @@ public ConnectorMetadata getMetadata(ConnectorTransactionHandle transactionHandl
     }
 
     private static class NoOpMetadata
-            implements ConnectorMetadata {}
+            implements ConnectorMetadata
+    {
+    }
+
+    private static class NoOpGroupProviderFactory
+            implements GroupProviderFactory
+    {
+        @Override
+        public String getName()
+        {
+            return "noop";
+        }
+
+        @Override
+        public GroupProvider create(Map<String, String> config)
+        {
+            return user -> Collections.emptySet();
```

Comment:
```suggestion
            checkArgument(config.isEmpty(), "noop group provider does not accept configuration properties");
            return user -> Collections.emptySet();
```

Reply:
actually:

`if (!config.isEmpty()) { ...` -- we don't have guava

and different wording:
```
"this group provider accepts no configuration properties"
```

see https://github.com/prestosql/presto/pull/3140